client/apps/hello_world/app.jsx

```diff
@@ -18,34 +19,44 @@ import './styles/styles';
 // Polyfill es6 promises for IE
 es6Promise.polyfill();
 
-class Root extends React.PureComponent {
+class Root extends React.Component {
```

Comment:
The reason i extracted this out into a new component is because the redux throws a fit if you try to hot reload the provider component. So hot reloading starts at this new root component versus the one with the provider in it